### ISSUE TITLE
no matches for kind "Config"

### ISSUE THREAD

**Issue by phoenixwu0229** [kubesphere/kubesphere#4447]
**What's your question**
install failed, ks-controller CrashLoopBackOff

**Environment: OS & Hardware Information**

k8s v1.22

**Error logs or message (Attach logs or screenshot)**

I1116 00:55:15.053713       1 clusterrolebinding_controller.go:98] Setting up event handlers
I1116 00:55:15.053797       1 globalrole_controller.go:99] Setting up event handlers
I1116 00:55:15.053844       1 globalrolebinding_controller.go:106] Setting up event handlers
I1116 00:55:15.054326       1 groupbinding_controller.go:97] Setting up event handlers
I1116 00:55:15.054377       1 group_controller.go:105] Setting up event handlers
I1116 00:55:15.106873       1 notification_controller.go:89] Setting up event handlers
E1116 00:55:15.113761       1 notification_controller.go:113] get /, Kind= informer error, no matches for kind "Config" in version "notification.kubesphere.io/v2beta1"
F1116 00:55:15.113806       1 server.go:340] unable to register controllers to the manager: no matches for kind "Config" in version "notification.kubesphere.io/v2beta1"

**Installer Version**

v3.2.0

**Comment by xyz-li** [comment]
Have you updated the image of ks-installer?

**Comment by iawia002** [comment]
Someone encountered the same problem on slack, here is the error message from ks-installer:

```
rendered manifests contain a resource that already exists. Unable to continue with install: FederatedTypeConfig "configmaps" in namespace "kube-federation-system" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "kubefed"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "kube-federation-system"
```

@pixiake can you take a look?

**Comment by pixiake** [comment]
You can check the logs of ks-installer by the command. 
```
kubectl logs -n kubesphere-system $(kubectl get pod -n kubesphere-system -l app=ks-install -o jsonpath='{.items[0].metadata.name}') -f
```

**Comment by jikehaohao** [comment]
I have the same problem,I installed this and solved the problem！

**Comment by jikehaohao** [comment]
Just install notification manager, but my version is v3 2.1. This notification manager is available in v3.1 Only before version 0.0, so I doubt whether it is v3.0 Version 2.1 also uses the functions of the previous version

**Comment by oldthreefeng** [comment]
the same issue.
